docs/core/additional-tools/dotnet-core-uninstall.md

```diff
@@ -0,0 +1,357 @@
+---
+title: .NET Core Uninstall Tool
+description: An overview of the .NET Core Uninstall Tool, a guided tool that enables the controlled clean-up of .NET Core SDKs and runtimes.
+author: sfoslund
+ms.date: 11/12/2019
+---
+# .NET Core Uninstall Tool
+
+The [.NET Core Uninstall Tool](https://github.com/dotnet/cli-lab/releases) (`dotnet-core-uninstall`) lets you clean up .NET Core SDKs and runtimes on a system such that only the desired versions remain. A collection of options is available to specify which versions you want to uninstall.
+
+The tool supports Windows and macOS. Linux is currently not supported.
+
+Note that the Windows version of this tool can only uninstall .NET Core SDKs and runtimes that were installed using .NET Core SDK and runtime installers. Prior to Visual Studio 2019 Update 16.3, the Visual Studio installer called the .NET Core SDK installer and these versions can be uninstalled with the tool. The `dotnet-core-uninstall list` command displays which SDKs can be uninstalled with the tool.
+
+## Install the tool
+
+You can download the .NET Core Uninstall Tool from [GitHub Releases](https://github.com/dotnet/cli-lab/releases).
+
+> [!NOTE]
+> The tool requires elevation to uninstall .NET Core SDKs and runtimes. Therefore, it should be installed in a write-protected directory such as *C:\Program Files* on Windows or */usr/local/bin* on macOS. See also [Elevated access for dotnet commands](../tools/elevated-access.md). Detailed installation instructions are available on the [GitHub Releases page](https://github.com/dotnet/cli-lab/releases).
+
+## How to use the tool
+
+1. Display installed .NET Core SDKs and runtimes that can be removed by this tool using `dotnet-core-uninstall list`.
```

Comment:
it felt weird to have install and uninstall steps in the how to use the tool guide.

Also, I'm wondering if each one of these items should be simplified and be a link to the section below or if we should simply skip and have the steps directly in the header such as in https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019

I'm also thinking that usage instructions and command syntax should be broken down into different sections.

Reply:
Having the steps directly in the header would make sense if we're splitting the options for `dotnet-core-uninstall dry-run` and `dotnet-core-uninstall remove` into two different sections. Is that what you mean by braking down the usage instructions or did you have something else in mind?

Reply:
I think install, use, and uninstall should be separate instructions instead of having install and uninstall as steps of how to use the tool - that seems wrong. I tried to make that more clear in the latest updates - see what you think! But I think what I'm also struggling with on this particular article is that is a mix of reference (with the commands syntax) and how to. 
I see two options:
- We merge as is (if you're ok with my changes) and watch for customer's feedback.
- We separate the commands syntax and the prescriptive guidance into different sections (or even different articles). We could even merge this and then I open a new PR with a proposal on top of this.

Reply:
It's up to you, but the changes you made look good to me and I am okay with merging this if @KathleenDollard is.